scripts/setup-updates

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/python3
+
+import sys
+import os
+import pydbus
+
+def symlink_updates(dest_dir, update_dir):
```

Comment:
Indentation 2 lines for top level functions please.

Reply:
Changed.